### DISCUSSION TITLE
global scope not working on edit/update on Nova 3?

### DISCUSSION THREAD

**Discussion by Sohail-purelogics** [laravel/nova-issues#4729] [Q&A] [answered]
I've multi-tenant application for which I've global scopes but these global scopes do not respect edit and update functionality.
I was able to make it work on detail view by overriding detailquery method, how can make it work on edit and update?

**Comment by crynobone** [comment]
There's `editQuery()` and `replicateQuery()` available for such usage.

**Reply by Sohail-purelogics** [reply]
I tried registering global scope in editQuery and replicateQuery methods in nova resource but still, I was able to get data without global scope

**Reply by Sohail-purelogics** [reply]
`public static function editQuery(NovaRequest $request, $query)
    {
        return $query->getModel()->registerGlobalScopes($query);
    }`

**Reply by crynobone** [reply]
Nova 4?

**Reply by Sohail-purelogics** [reply]
Nova 3

**Reply by crynobone** [reply]
Your only options for Nova 3 is to override and rebind `Laravel\Nova\Http\Controllers\UpdateFieldController` controller to the Laravel Container

**Reply by Sohail-purelogics** [reply]
Could you please share details how to do that?

**Reply by crynobone** [reply]
Here one example for different use case: https://github.com/nova-issues/issue-4516/commit/2498ae7693587fb516ad6447d7fcab39903591eb

**Reply by guiribmedeiros** [reply]
Hi @crynobone,
Is there any better way for that exact scenario, but with Nova 4?

**Comment by Sohail-purelogics** [comment, marked as answer]
Thanks, @crynobone Override and rebind Laravel\Nova\Http\Controllers\UpdateFieldController controller to the Laravel Container works for me.
